environment-dev.yml

```diff
@@ -2,7 +2,7 @@ name: modin
 channels:
   - conda-forge
 dependencies:
-  - pandas==1.5.1
+  # - pandas==1.5.2
```

Comment:
conda hasn't picked up yet the release.

Reply:
Can we merge the PR like this, or should we wait until conda has picked it up?

Reply:
It seems that is a conda issue in GH actions. I can install pandas 1.5.2 locally with conda. I encountered the same problem in #5059 when installing unidist-mpi 0.2.1. I think it is okay to install pandas from PyPI in dev env files for now but we should definetely figure out why conda can't install latest versions of pandas and unidist in GH actions.

Reply:
IMHO this should be investigated instead of playing an ostrich shoving the head beneath some sand.